### PR TITLE
Updating OF docs & example

### PR DESCRIPTION
**Pull request by szalpal**

### CODE REVIEW

**Comment by szalpal** [comment]
!build

**Comment by szalpal** [comment]
!build

File: docs/examples/optical_flow/optical_flow_example.ipynb [hunk 1]
```diff
@@ -4,30 +4,22 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Optical flow example\n",
-    "This is an example of caluclating Optical flow using Dali pipeline."
+    "# Optical Flow example\n",
+    "This notebook presents, how to use Dali to calculate optical flow for given sequence of frames."
```
Comment (Kh4L) [inline docs/examples/optical_flow/optical_flow_example.ipynb]:
```suggestion
    "This notebook presents how to use Dali to calculate optical flow for given sequence of frames."
```
I think you don't need a comma here

File: docs/examples/optical_flow/optical_flow_example.ipynb [hunk 2]
```diff
@@ -36,64 +28,43 @@
...
-    "print(\"snippet 1\")"
+    "from matplotlib import pyplot as plt"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Setting metaparamteres"
+    "Setting metaparamteres.  \n",
```
Comment (Kh4L) [inline docs/examples/optical_flow/optical_flow_example.ipynb]:
```suggestion
    "Setting metaparameters.  \n",
```

File: docs/examples/optical_flow/optical_flow_example.ipynb [hunk 3]
```diff
@@ -247,29 +208,26 @@
...
-    "print(\"snippet 4\")"
+    "        return of"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Build and run"
+    "### Build and run Dali Pipeline."
```
Comment (Kh4L) [inline docs/examples/optical_flow/optical_flow_example.ipynb]:
```suggestion
    "### Build and run DALI Pipeline."
```

File: docs/examples/optical_flow/optical_flow_example.ipynb (hunk 1 above)
Comment (szalpal) [reply docs/examples/optical_flow/optical_flow_example.ipynb]: In polish, I would put it there...

**Pull request merged**